src/libs/portfolio/portfolio.ts

```diff
@@ -246,7 +251,10 @@ export class Portfolio {
       )
     ])
 
-    const [tokensWithErrResult, blockNumber, beforeNonce, afterNonce] = tokensWithErr
+    console.log(`tokensWithErr after flattening on network ${networkId}:`, tokensWithErr)
```

Comment:
is the console.log intentional? we shouldn't have console logs in libraries imo, unless it's on errors

Reply:
Its only for testing before merge, will remove it